radio/src/translations/fr.h

```diff
@@ -191,7 +191,7 @@
 #define TR_SRC_BATT                    "Batt"
 #define TR_SRC_TIME                    "H_M"
 #define TR_SRC_GPS                     "GPS"
-#define TR_SRC_TIMER                   "Chr"
```

Comment:
@3djc Where would be the smallest space this is shown on B&W? Just wondering if we're limited to 4 characters or not. I'm thinking perhaps the LS view on 128x64 is about the smallest, and "Time" fits there, with space for perhaps one more character, although it would look crammed then.

Reply:
Yeah, sources should not go above 4 letters (for example, telem sensors name are limited to 4 too)